### PR TITLE
Fixes #36204 - Refine empty states for CV UI

### PR DESCRIPTION
**Pull request by sjha4**
#### What are the changes introduced in this pull request?

Make empty states xl.
Update text on some of the empty state messages we show.

#### What are the testing steps for this pull request?
Test out empty states on CV UI pages.
Check if they make sense. :)

### CODE REVIEW

File: webpack/scenes/ContentViews/Details/ComponentContentViews/ContentViewComponents.js [hunk 1]
```diff
@@ -216,8 +216,8 @@ const ContentViewComponents = ({ cvId, details }) => {
     </DropdownItem>,
   ];
 
-  const emptyContentTitle = __(`No content views belong to ${label}`);
-  const emptyContentBody = __('Please add some content views.');
+  const emptyContentTitle = __('No content views to add yet');
+  const emptyContentBody = __('Please create some component content views.');
```
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Details/ComponentContentViews/ContentViewComponents.js:220]: We don't use the term "component content views" any more..

```suggestion
  const emptyContentBody = __('Please create some content views.');
```

File: webpack/scenes/ContentViews/Details/Filters/CVErrataIDFilterContent.js [hunk 2]
```diff
@@ -239,8 +237,8 @@ const CVErrataIDFilterContent = ({
     dateType === 'issued' &&
     statusSelected === ALL_STATUSES;
 
-  const emptyContentTitle = __('No errata available to add to this filter.');
-  const emptyContentBody = __('No errata available for this content view.');
+  const emptyContentTitle = __('No errata filter rules yet');
+  const emptyContentBody = __('Add repositories with errata to content view to select them here.');
```
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Details/Filters/CVErrataIDFilterContent.js:241]: I don't understand this. Is there a clearer way to word it?

File: webpack/scenes/ContentViews/Details/Filters/CVModuleStreamFilterContent.js [hunk 3]
```diff
@@ -194,8 +194,8 @@ const CVModuleStreamFilterContent = ({
     },
   ];
 
-  const emptyContentTitle = __('No rules have been added to this filter.');
-  const emptyContentBody = __("Add to this filter using the 'Add filter rule' button.");
+  const emptyContentTitle = __('No module stream filter rules yet');
+  const emptyContentBody = __('Add repositories to content view with module streams to select them here.');
```
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Details/Filters/CVModuleStreamFilterContent.js:198]: Same here.. Is the message about not having module stream filter rules? Or not having module streams in the content view?

**Review by jeremylenz — COMMENTED** [review]
Thanks @sjha4!

Some comments below

File: webpack/scenes/ContentViews/Details/Filters/CVModuleStreamFilterContent.js (hunk 3 above)
Comment (jeremylenz) [reply webpack/scenes/ContentViews/Details/Filters/CVModuleStreamFilterContent.js:198]: Shouldn't emptyContentTitle and emptyContentBody match, then?

File: webpack/scenes/ContentViews/Details/Filters/CVErrataIDFilterContent.js (hunk 2 above)
Comment (sjha4) [reply webpack/scenes/ContentViews/Details/Filters/CVErrataIDFilterContent.js]: Updated to this:
```
const emptyContentTitle = __('No errata filter rules yet');
const emptyContentBody = __('No errata to add yet');
```

**Review by jeremylenz — APPROVED** [review]
Thanks @sjha4 

LGTM :+1:

**Pull request merged**